modules/introduction/pages/editions.adoc

```diff
@@ -1,38 +1,32 @@
-The release version of CE is several months behind EE.
-CE also does not include the latest bug fixes.
-Documentation, mailing lists, and forums provide support for the Couchbase user community to help troubleshoot issues and answer questions.
-CE comes with limited concurrency and parallelism and supports a maximum of 4 cores per node.
-
-Open source project::
-The Couchbase Server open source project is a platform for innovation.
-Couchbase is committed to open source development and the open source project continues to serve as the foundation for both the Community Edition and the Enterprise Edition.
-There are many ways to contribute to the open source project: You can contribute code to our core engine, to our SDKs, connectors and other integration components that help us connect to other products through https://github.com/couchbase[github.com/couchbase^], you can report issues on our tracking system: https://issues.couchbase.com/projects/MB?selectedItem=com.atlassian.jira.jira-projects-plugin:release-page[JIRA^], and https://docs.couchbase.com/home/contribute/index.html[contribute to the documentation] either by submitting changes on GitHub or by simply clicking the "Feedback" link in our online documentation.
-The full details on contributing to open source can be found on the open source page http://developer.couchbase.com/open-source-projects[developer.couchbase.com/open-source-projects^].
+
+One important principle for CE and EE is to ensure full portability of applications between two editions. The capabilities ensure that an application running on EE should simply transition to CE without code changes and visa versa.
+
+CE comes free, and there are no constraints on using these binaries in production systems. However, CE is not subjected to the iterative "test, fix, and verify" quality assurance cycle that is a part of the Enterprise Edition release process. The release version of CE is several months behind EE. CE also does not include the latest bug fixes.
+
+Documentation, mailing lists, and forums support the Couchbase user community to help troubleshoot issues and answer questions. CE comes with limited concurrency and parallelism and supports a maximum of 4 cores per node.
+
+Open Source Project::
+The Couchbase Server open source project is a platform for innovation. Couchbase is committed to open source development, and the open source project continues to serve as the foundation for both the Community Edition and the Enterprise Edition.
+
+There are many ways to contribute to the open source project:
+* Contribute code to our core engine, to our SDKs, connectors and other integration components that help us connect to other products through github.com/couchbase. Find more details on the Open Source Projects page.
+* Report issues on our tracking system: JIRA.
+* Contribute to the documentation either by submitting changes on GitHub or by simply clicking the "Feedback" link in our online documentation.
```

Comment:
1. Needs a line break before the start of the bulleted list, otherwise they just show up as asterisks.
2. Links as specified in the original Word doc.

```suggestion
There are many ways to contribute to the open source project:

* Contribute code to our core engine, to our SDKs, connectors and other integration components that help us connect to other products through https://github.com/couchbase[^].
Find more details on the http://developer.couchbase.com/open-source-projects[Open Source Projects^] page.
* Report issues on our tracking system: https://issues.couchbase.com/projects/MB?selectedItem=com.atlassian.jira.jira-projects-plugin:release-page[JIRA^].
* xref:home:contribute:index.adoc[Contribute to the documentation] either by submitting changes on GitHub or by simply clicking the "Feedback" link in our online documentation.
```